app/main.py

```diff
@@ -1,8 +1,57 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
```

Comment:
Apply formatter, you can try black here

Reply:
> still not fixed indentation

Can you add more details, pls?
In my opinion, the return type ") -> None:" should be either at the end of the 5th line or on the 6th line in alignment with "def".
But in the first case occurs  flake8 ".\main.py:2:80: E501 line too long (80 > 79 characters)" 
and in the second ".\main.py:3:5: E124 closing bracket does not match visual indentation".

Reply:
or should I place each __init__ parameter from a new line?